### DISCUSSION TITLE
Sensor WAN Internet

### DISCUSSION THREAD

**Discussion by milandzuris** [Vaskivskyi/ha-asusrouter#730] [Q&A]
Hi i have question how work Sensor WAN and Internet whats the diffrence between this Sensors?

**Comment by Vaskivskyi** [comment]
There are now the following entities:

- `binary_sensor.{}_internet` - showing whether your device is connected to the web, with attributes `ip_address` and `wan_unit`. The last shows whether the main WAN is connected (`0`) or the secondary USB (`1`)
- `binary_sensor.{}_wan` and `.{}_wan_secondary` displaying the connection status of the corresponding interface. They also have `link` and `primary` attributes which you see now. `link` indicates whether this interface is physically connected (e.g., ethernet cable or USB modem plugged), `primary` shows whether this interface is used for the connection to the web (`internet`)

So, in short - `internet` just shows whether your device is connected. `wan` shows details of the exact connection way.
